requirements/requirements_lightning.txt

```diff
@@ -1,6 +1,6 @@
 hydra-core>=1.2.0,<1.3
 omegaconf>=2.2,<2.3
-pytorch-lightning>=1.8.6
+pytorch-lightning>=1.8.6,<1.9.0
```

Comment:
Temporary fix CI :)

Reply:
I think fix got merged, rebase and revert this

Reply:
Fixed